### PR TITLE
fix typo in WTrackProperty

### PR DESCRIPTION
**Pull request by ronso0**
**edit** after merging 2.4 with the other bugfix that uses the same mechanics, this is now just a typo fix.
_____

one more fix for good track menu / tracks view UX.

* load a track from current view
* deck track menu (via CO / controler) -> move track file to trash -> Yes
* (deck is stopped, track ejected)
* track view is repopulated (and eventually resorted)
* **currently:** current index is reset, next move jumps to top left index
* **this PR:** prev. selection and current index is restored (selection off by one track)

### Note:
~~When testing with a playlist view you'll hit #13111.~~ (can now reproduce only in `main`/2.5-alpha :face_with_spiral_eyes: )

### CODE REVIEW

File: src/widget/wtrackproperty.cpp [hunk 1]
```diff
@@ -121,4 +121,15 @@ void WTrackProperty::ensureTrackMenuIsCreated() {
         m_pTrackMenu = make_parented<WTrackMenu>(
                 this, m_pConfig, m_pLibrary, WTrackMenu::kDeckTrackMenuFeatures);
     }
+    // Before and after the loaded tracks file has been removed from disk,
+    // instruct the library to save and restore the current inde for
```
Comment (m0dB) [inline src/widget/wtrackproperty.cpp:125]: typo, index

**Comment by m0dB** [comment]
LGTM

Note that this PR targets 2.4. Shouldn't that be main?

**Comment by ronso0** [comment]
2.4 is correct.
I mentioned deck menu via CO, but I noticed that's irrelevant. The the view is also not updated when a track is purged via the deck menu.

**Review by daschuer — COMMENTED** [review]
Now this PR is melted down to be only a typo fix. 
Is that correct? Can you adjust the title then?

**Review by daschuer — APPROVED** [review]
Thank you LGTM

**Pull request merged**